tests/unit/badge/__snapshots__/badge.spec.js.snap

```diff
@@ -53,7 +53,7 @@ exports[`Badge should return content inside container if title not provided 1`]
 </div>
 `;
 
-exports[`Badge should return noscript if children and title not provided 1`] = `<noscript />`;
+exports[`Badge should return noscript if children and title not provided 1`] = `""`;
```

Comment:
You could test `return null` cases without snapshot. Just do this: `expect(component.type()).toEqual(null);`

Reply:
Oh, I see you changed that. So that way should work here as well, right?

Reply:
Added assertion to verify that it returns `null`.

Reply:
Cool, thanks! ...but do we still really need to compare to snapshot? I would suggest to not create snapshots in these cases and just check the `type()`/`html()`. How do you think guys?

Reply:
Strangely, the `type()` did not work... Don't know why, though.

Reply:
Removed